src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -732,4 +733,28 @@ private void discoverModCompat()
             Compatibility.dynamicTreesCompat = new DynamicTreeCompat();
         }
     }
+
+    /**
+     * Gets all the possible foods for the cook.
+     * 
+     * @return a set of the foods.
+     */
+    public static Set<ItemStorage> getAllFoods()
+    {
+         Set<ItemStorage> edibles = IColonyManager.getInstance().getCompatibilityManager().getEdibles();
+         edibles.removeIf(item -> ISCOOKABLE.test(item.getItemStack()));
```

Comment:
I double checked in the code, this is all not necessary, IColonyManager.getInstance().getCompatibilityManager().getEdibles() is enough, no need for the filter.

Reply:
OK, I thought "It was there before, I'm sure theres a reason", but I'll remove it then.